### ISSUE TITLE
Create a basic dashboard

### ISSUE THREAD

**Issue by antonmc** [votesavvyrhok/votesavvy#8]
Floating this idea - it might be useful to provide a basic dashboard - at least a page that shows a count and rate of survey completion by young voters - accessible via a devoted url, and protected by some form of authentication - so that the VoteSavvy team can check progress.

**Comment by jaydez** [comment]
Great idea. This might work well with having the API to access voter results - we could also add stats to the database like the ones you mention as surveys come in, which can then be queried.

**Comment by yingqiaojian** [comment]
I agree with Jonathan. The dashboard could be the front end. The data aggregation function could be at the back end.

**Comment by yingqiaojian** [comment]
I may use the D3.js library to draw the figures of data
